content/english/java-basics/activity-7.md

```diff
@@ -1,6 +1,126 @@
 ---
-title: "Data Structure"
+title: "Data Structure (Array & ArrayList)"
 description: "Introduce basic data structures in Java."
 date: 2020-07-16T00:00:00Z
 weight: 8
----
\ No newline at end of file
+---
+In previous exercises, we learned that variables allow us to access and store a piece data. But, what if we want to store a bunch of similiar data? Is there a good way to organize them without declaring multiple variables?
+
+In programming, <b>data structure</b> are structures that allowed coders to store multiple pieces of data of the same type. Using data structure is an efficient way to store and access large pieces of data. In this exercise, we will learn about 2 data structures used in Java: <b>array</b> and <b>ArrayList</b>.
```

Comment:
I think it would be helpful to quickly explain the difference between arrays and ArrayLists.

Reply:
added!